packages/app-bridge/src/react/useGuidelineAssetsSettings.ts

```diff
@@ -0,0 +1,26 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useState } from 'react';
+import { GuidelineAssetsSettings } from '../types/GuidelineAssetSettings';
+import { Handler } from 'mitt';
+
+export const useGuidelineAssetsSettings = () => {
+    const [assetSettings, setAssetSettings] = useState<GuidelineAssetsSettings>({
```

Comment:
Nit: as we call the hook `useGuidelineAssetsSettings` I would also call the state `assetsSettings`

```suggestion
    const [assetsSettings, setAssetsSettings] = useState<GuidelineAssetsSettings>({
```